src/lammps.cpp

```diff
@@ -442,7 +442,11 @@ LAMMPS::LAMMPS(int narg, char **arg, MPI_Comm communicator) :
       iarg += 3;
       while (iarg < narg && arg[iarg][0] != '-') iarg++;
 
-    } else error->universe_all(FLERR,"Invalid command-line argument");
+    } else {
+      std::string errmsg("Invalid command-line argument");
+      errmsg += arg[iarg];
+      error->universe_all(FLERR,errmsg.c_str());
```

Comment:
```suggestion
      error->universe_all(FLERR,errmsg);
```

Alternately, it should be possible to use just a single line like this:
 `error->universe_all(FLERR, fmt::format("Unknown pair style mliap argument: {}", arg[iarg]));`

Reply:
Went with alternate.  cleaner.  Old habits die hard.